packages/php/email-editor/composer.json

```diff
@@ -47,7 +47,7 @@
 		"test:integration": "wp-env run tests-cli --env-cwd=wp-content/plugins/email-editor ./vendor/bin/phpunit --configuration phpunit-integration.xml.dist",
 		"phpcs": "phpcs -s -p",
 		"phpcbf": "phpcbf -p",
-		"phpstan": "php ./tasks/run-phpstan.php"
+		"phpstan": "vendor/bin/phpstan analyse -c phpstan.neon"
```

Comment:
I would suggest we add a parameter for memory to the command

```
"phpstan": "vendor/bin/phpstan analyse -c phpstan.neon --memory-limit=2G"
```

This will avoid memory limit errors when someone has PHP configured with a strict memory limit locally.